xml/Microsoft.Build.BuildEngine/LocalNode.xml

```diff
@@ -31,7 +31,7 @@
         <AssemblyVersion>4.0.0.0</AssemblyVersion>
       </AssemblyInfo>
       <ReturnValue>
-        <ReturnType>System.Void</ReturnType>
+        <ReturnType FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">System.Void</ReturnType>
```

Comment:
I'm going to double check some of these results ... cause I'm not sure why it's adding the attribute here. 

@BillWagner can you verify whether this was run against the binaries repo's `master` branch?

Reply:
@joelmartinez Yes, it was run against master. But, master has some in-progress work that hasn't published for the MS build updates. (@ghogen and @v-alje are working on those)

Reply:
@joelmartinez the latest change removed many FrameworkAlternate attributes, but this one on the <ReturnType> is still there.

Reply:
@TianqiZhang oh I think this might just be a confusing way of the github UX showing. It says the diff is outdated. You can see the latest one here that shows the FXA attribute removed :)
https://github.com/dotnet/dotnet-api-docs/pull/4303/commits/f555a86b5d08d93daf39c48d3328ba6c09510c15#diff-5042876fc5d6b9fbafbc9a53dc65c984R34